android/src/main/kotlin/com/onegini/mobile/sdk/flutter/handlers/CustomRegistrationHandler.kt

```diff
@@ -0,0 +1,14 @@
+package com.onegini.mobile.sdk.flutter.handlers
+
+import com.onegini.mobile.sdk.flutter.providers.CustomRegistrationAction
+
+class CustomRegistrationHandler {
+  fun getCustomRegistrationAction(
+    idProvider: String?,
+    customRegistrationActions: ArrayList<CustomRegistrationAction>
+  ): CustomRegistrationAction? {
+    if (idProvider == null) return null
```

Comment:
this can be one liner
`return idProvider?.let { customRegistrationActions.find { it.getIdProvider() == idProvider } }`
one liners in Kotlin can be retuned using `=` though I would follow the pattern that is already used in the codebase.

Reply:
done with the one line